.github/workflows/dispatch-k6-performance.yml

```diff
@@ -0,0 +1,60 @@
+name: Run K6 performance test
+
+on:
+  workflow_dispatch:
+    inputs:
+      apiVersion:
+        description: 'API Version'
+        required: true
+        default: 'v1'
+      environment:
+        description: 'Environment'
+        required: true
+        default: 'staging'
+        type: choice
+        options:
+          - test
+          - staging
+          - performance
+      tokens:
+        description: 'Tokens to generate; for create dialog, search, none, or both'
+        required: true
+        default: 'both'
+        type: choice
+        options:
+          - both
+          - enterprise
+          - personal
+          - none
+      vus:
+        description: 'Number of VUS'
+        required: true
+        type: number
+        default: 10
+      duration:
+        description: 'Duration of test, ie 30s, 1m, 10m'
+        required: true
+        default: 1m
+        type: string
+      testSuitePath:
+        description: 'Path to test suite to run'
+        required: true
+        default: 'tests/k6/tests/serviceowner/performance/createremove-no-delay.js'
```

Comment:
Hm, this default file does not exist? 🤔

Reply:
It does exist, but should not be default